tests/js-tests/main.js

```diff
@@ -117,7 +117,15 @@ cc.game.onStart = function(){
     }
     else
     {
-        cc.loader.resPath = '../cpp-tests/Resources'
+        // js-test use cpptest resource in debug mode , and in the release mode, console will copy the resource into the res dir
+        // so the respath will modify to res,
+        if (cc.game.config[cc.game.CONFIG_KEY.engineDir] !== "frameworks/cocos2d-html5") {
```

Comment:
I know it's this value today, but who can say next month we won't change it, so please modify it to something more general